### DISCUSSION TITLE
Lidar

### DISCUSSION THREAD

**Discussion by namazmunira** [ApolloAuto/apollo#15006] [Q&A]
hello, please tell me, is it possible to use ouster and c32w lidar instead of Velodyne? what should be taken into account? what are the requirements ?

**Comment by daohu527** [comment]
@namazmunira Not ready, ouster currently not supported. leishen c32w lidar is supported now!

We will refer to the ros driver to transplant the implementation.
https://github.com/ouster-lidar/ouster-ros
